app/models/customer.rb

```diff
@@ -0,0 +1,3 @@
+class Customer < ApplicationRecord
```

Comment:
I guess I always do it the other way (validates presence: true)

This obviously works as well, but we may want to be consistent.